docs/mkdocs.yml

```diff
@@ -86,9 +91,13 @@ plugins:
       default_handler: python
       handlers:
         python:
-          rendering:
+          options:
             show_source: true
-            heading_level: 3
+            heading_level: 2
+            show_root_toc_entry: true
+            show_symbol_type_toc: true
+            show_root_full_path: false
+            show_root_heading: true
```

Comment:
How do you feel about adding?
```
show_source: true
merge_init_into_class: true
```

Reply:
Show source is already in I think? For `merge_init_into_class: true`, I am not sure, I like the intro + constructor style but both are possible, depends on the preference here :)

Reply:
Haha yeah, I must have overlooked it :man_facepalming: